samples/client/petstore/kotlin-multiplatform-new/src/common/main/org/openapitools/client/models/Order.kt

```diff
@@ -0,0 +1,60 @@
+/**
+ * OpenAPI Petstore
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+@file:Suppress("UnusedImport", "RemoveRedundantQualifierName")
+
+package org.openapitools.client.models
+
+
+import kotlinx.serialization.Required
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * An order for a pets from the pet store
+ *
+ * @param id 
+ * @param petId 
+ * @param quantity 
+ * @param shipDate 
+ * @param status Order Status
+ * @param complete 
+ */
+@Suppress("ArrayInDataClass")
+@Serializable
+public data class Order(
+    @SerialName(value = "id")
+    val id: kotlin.Long? = null,
+    @SerialName(value = "petId")
+    val petId: kotlin.Long? = null,
+    @SerialName(value = "quantity")
+    val quantity: kotlin.Int? = null,
+    @SerialName(value = "shipDate")
+    val shipDate: kotlinx.datetime.LocalDateTime? = null,
```

Comment:
Using the default `dateLibrary`, or specifying `string`, still maps `date-time` to `kotlinx.datetime.LocalDateTime`. This results in an invalid model that does not compile because of a missing `UseSerializers` - but there should be no reference to `kotlinx.datetime` at all, the generated property should be `val shipDate: String? = null`.

I've run into this same issue running this generator locally, but have not managed to pin down the issue yet.

Reply:
Interesting...

Reply:
Yeah I was writing too much kotlin, forgot a `break`

Reply:
Is switched into the `STRING` case, but fell through and updated everything to `kotlinx`

Reply:
Ah, nice find! Reading too much Kotlin myself too 😄